python/hsfs/feature_view.py

```diff
@@ -2128,6 +2124,303 @@ def delete_all_training_datasets(self):
         """
         self._feature_view_engine.delete_training_data(self)
 
+    def _get_feature_monitoring_configs(
```

Comment:
should have a `get_scheduled_statistics_monitoring_configs` also? users may not know they the config id are shared between statistics config and fm config.

Reply:
Ja that is one of the question. Maybe we could rename it to monitoring config to implicitly cover both cases?